deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +954,27 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        request_url = "http://"
+        if self._arg_parser.rgw_tls_cert_path:
+            request_url = "https://"
+        request_url = request_url + rgw_endpoint + "/admin/info" + "?" + "format=json"    
```

Comment:
Doesn't Python have a lib to build URI? Instead of the "manual" string concat. The equivalent of go `url.Values{}` from `"net/url"`.